src/ui/Pricing/SubscriptionSettings/index.svelte

```diff
@@ -179,12 +179,64 @@
     gap: 16px;
   }
 
+  .overview {
+    margin-top: 6px;
+  }
+
+  .txt-right .btn-2:not(:first-child) {
+    margin-left: 12px;
+  }
+
   :global(.phone),
   :global(.phone-xs) {
     .btn-1 {
       width: 100%;
       margin-top: 8px;
     }
+
+    .card {
+      --margin: 6px;
+    }
+
+    .payment-method {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
```

Comment:
Repeated rules across multiple classes within one component. It can be abstracted to one class